cmd/kube-scheduler/app/server.go

```diff
@@ -157,9 +157,6 @@ func runCommand(cmd *cobra.Command, args []string, opts *options.Options) error
 
 // Run executes the scheduler based on the given configuration. It only return on error or when stopCh is closed.
 func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}) error {
-	// To help debugging, immediately log version
-	klog.V(1).Infof("Starting Kubernetes Scheduler version %+v", version.Get())
```

Comment:
It is logged already at L142

Reply:
L142 is not as complete and does not have verbosity. Could you remove L142 instead, or better replace L142 with this line.

Reply:
OK

Reply:
@bsalamat   please add a tag